src/constructs/iam/policies/parameter-store-read.ts

```diff
@@ -8,8 +8,8 @@ export class GuParameterStoreReadPolicyStatement extends PolicyStatement {
   constructor(scope: GuStack, props: AppIdentity) {
     super({
       effect: Effect.ALLOW,
-      actions: ["ssm:GetParametersByPath"],
-      resources: [`arn:aws:ssm:${scope.region}:${scope.account}:parameter/${scope.stage}/${scope.stack}/${props.app}`],
+      actions: ["ssm:GetParametersByPath", "ssm:GetParameters", "ssm:GetParameter"],
+      resources: [`arn:aws:ssm:${scope.region}:${scope.account}:parameter/${scope.stage}/${scope.stack}/${props.app}*`],
```

Comment:
Should this be `/*`?

Reply:
> presumably changing to `/*` would tighten this up?

Yeah, it adds more namespacing. My assumption is we have parameters like `/PROD/deploy/amigo/playSecret` and not `/PROD/deploy/amigo-playSecret`?

Reply:
Yes, I think you're right - I'll update this

Reply:
Unfortunately https://github.com/guardian/cdk/pull/1076/commits/1b93f2d491fd032e4c90d7cf8f5035f1de4495c2 does not work as desired. A [deployment of Amigo (based on this change) fails](https://riffraff.gutools.co.uk/deployment/view/696519e8-e935-4d19-a47c-8a4d6bf4aee7) with the following error:

`User: arn:aws:sts::<redacted>:assumed-role/amigo-CODE-InstanceRoleAmigo75944747-1WWFSO5Q5QL8F/i-<redacted> is not authorized to perform: ssm:GetParametersByPath on resource: arn:aws:ssm:eu-west-1:<redacted>:parameter/CODE/deploy/amigo because no identity-based policy allows the ssm:GetParametersByPath action` 

It seems that adding the `/` means that the permissions no longer match the request that `simple-configuration` is making.

I think the only way that we can achieve the level of precision that we want is by using two separate policy statements: https://github.com/guardian/cdk/pull/1076/commits/a9197fccd4ec921568e9086aa6d201c5bda5e1ff